libraries/aws_cloudtrail_trail.rb

```diff
@@ -0,0 +1,103 @@
+  
```

Comment:
This local resource seems to added by accident and not needed. Please remove it

Reply:
The local resource has been removed: [commit link](https://github.com/mitre/cis-aws-foundations-baseline/commit/cc6f575a036d2669671f8b15a90ddc180aafb217)